ci/test/00_setup_env_native_qt5.sh

```diff
@@ -15,5 +15,5 @@ export RUN_UNIT_TESTS_SEQUENTIAL="true"
 export RUN_UNIT_TESTS="false"
 export GOAL="install"
 export PREVIOUS_RELEASES_TO_DOWNLOAD="v0.15.2 v0.16.3 v0.17.2 v0.18.1 v0.19.1 v0.20.1"
-export BITCOIN_CONFIG="--enable-zmq --with-libs=no --with-gui=qt5 --enable-reduce-exports
+export BITCOIN_CONFIG="--enable-zmq --with-libs=no --with-gui=qt5 --enable-reduce-exports \
```

Comment:
Does this change change behavior?

Reply:
No, it doesn't. Why do you think so?

Reply:
Ok, was just wondering if there is a difference.

Reply:
More details.

Being quoted, the `"$BITCOIN_CONFIG` preserves the `newline` character.

Now it is [treated](https://www.gnu.org/software/bash/manual/html_node/Escape-Character.html#Escape-Character) as a line continuation.